src/main/java/de/tum/in/www1/artemis/repository/ExamRepository.java

```diff
@@ -329,7 +347,7 @@ WHERE TYPE(ex) = QuizExercise
      * @return true if the user is registered for the exam
      */
     @Query("""
-            SELECT CASE WHEN COUNT(exam) > 0 THEN true ELSE false END
+            SELECT CASE WHEN COUNT(exam) > 0 THEN TRUE ELSE FALSE END
```

Comment:
Would this work?
```suggestion
            SELECT COUNT(exam) > 0
```

Reply:
Let's try this

Reply:
Unless the e2e tests detect any problems, this seems to be valid